ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.cc

```diff
@@ -95,10 +95,13 @@ class TFCore
   tensor_type getTensorTypeFromTF (TF_DataType tfType);
   TF_DataType getTensorTypeToTF (tensor_type tType);
   int validateTensor (const GstTensorsInfo * tensorInfo, int is_input);
+  static void releaseBuffer (void* data, size_t t);
 };
 
-void init_filter_tf (void) __attribute__ ((constructor));
-void fini_filter_tf (void) __attribute__ ((destructor));
+extern "C" {
+  void init_filter_tf (void) __attribute__ ((constructor));
+  void fini_filter_tf (void) __attribute__ ((destructor));
```

Comment:
These two functions are defined in this .cc file; thus, the compiler will provide C++ call convention for them. Why do you need extern "C" for them?

Reply:
actually unnecessary now. I will revert these lines.